test-case/check-pause-release-suspend-resume.sh

```diff
@@ -51,7 +51,10 @@ set -e
 ##  * No unexpected errors should be present in dmesg during or after test
 ##      completion.
 
-source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+TOPDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")"/..; pwd)
+
+# shellcheck source=case-lib/lib.sh
+source "$TOPDIR"/case-lib/lib.sh
```

Comment:
Wait, this wasn't already done?

Reply:
It was on line 54? Not sure I understand the question sorry.